c7n/resources/account.py

```diff
@@ -1574,7 +1574,7 @@ def get_target(self, resource):
         resource[self.annotation].update(encryption_setting.get('ConnectionPasswordEncryption'))
 
         for kmskey in self.data:
-            if not self.data[kmskey].startswith('alias'):
+            if not isinstance(self.data[kmskey], str) or not self.data[kmskey].startswith('alias'):
```

Comment:
so this probably should have come up in review before, for compound words on variable names we use underscores between words, kms_key.

actually looking at the source here, this isn't really a kms_key at all.. its encrypt_attr .. also this should be explicitly skipping, the filter type.

Reply:
Addressed! I agree with you. Since, this logic was already there from the past, I just dumped it in a class!

Reply:
right, buts its logic you added in a previous pr on the same topic, hence the call out.

Reply:
please switch to looping over an *explicit* white list of known keys.

Reply:
Addressed! Thanks